### ISSUE TITLE
Docs: Add an example for Lambda Layers

### ISSUE THREAD

**Issue by jayair** [sst/sst#4532]

**Comment by DavideSegullo** [comment]
Hello @jayair , any updates related to this kind of example? Is there any reference? I saw this one: https://v2.sst.dev/advanced/lambda-layers but maybe it's too old

**Comment by jayair** [comment]
Yeah that's an old one. I didn't get a chance to get to it yet. What are you looking to use a Layer for?

**Comment by DavideSegullo** [comment]
> Yeah that's an old one. I didn't get a chance to get to it yet. What are you looking to use a Layer for?

Hey, ye I would like to use it for sharing node_modules between different lambda that actually use the same dependencies

**Comment by jayair** [comment]
If these are regular npm packages, then don't do that. https://aaronstuyvenberg.com/posts/why-you-should-not-use-lambda-layers
